docs/2_3_range-plots.fsx

```diff
@@ -52,8 +52,8 @@ let range1 =
     Chart.Range(
         x,y,yUpper,yLower,
         StyleParam.Mode.Lines_Markers,
-        Color="grey",
-        RangeColor="lightblue")
+        Color = Color.ColorString "grey",
```

Comment:
Shouldn't it be called `Color.fromString` for consistency?

Reply:
True. Thanks for the hint.